content/rs/installing-upgrading/_index.md

```diff
@@ -115,30 +100,41 @@ To install from the CLI:
 
 1. Copy the installation package to the node.
 
-    {{% expand "How do I know the package is authentic?" %}}
-For Ubuntu DEB packages and RHEL RPM packages, you can use the {{< download "GPG key file" "GPG-KEY-redislabs-packages.gpg" >}} to confirm authenticity of the package:
-
-- For Ubuntu:
-    1. Import the key with: `gpg --import <path to GPG key>`
-    1. Verify the package signature with: `dpkg-sig --verify <path to installation package>`
-- For RHEL:
-    1. Import the key with: `rpm --import <path to GPG key>`
-    1. Verify the package signature with: `rpm --checksig <path to installation package>`
-    {{% /expand %}}
+    Use the {{< download "GPG key file" "GPG-KEY-redislabs-packages.gpg" >}} to confirm authenticity of Ubuntu/Debian or RHEL RPM packages:
```

Comment:
This is optional, do we want to explicitly ask for it? It has some value, but also makes the process longer.
@adisht what do you think?

Reply:
Updated.